src/Workleap.Extensions.OpenAPI/TypedResult/ExtractSchemaTypeResultFilter.cs

```diff
@@ -121,6 +135,28 @@ internal static IEnumerable<ResponseMetadata> GetResponsesMetadata(Type returnTy
         return null;
     }
 
+    private static HashSet<int> ExtractResponseCodesFromAttributes(OperationFilterContext context)
```

Comment:
Could be interesting to add unit test, probably a way to pass the MethodInfo or something similar instead of the whole context for easier testing.